util/keyfile/src/bin/keygen_main.rs

```diff
@@ -3,29 +3,60 @@
 
 //! A CLI tool for generating individual MobileCoin identities
 
+use bip39::{Language, Mnemonic};
 use clap::Parser;
-use mc_account_keys::{RootEntropy, RootIdentity};
-use mc_util_keyfile::config::Config;
+use mc_util_keyfile::{config::Config as GeneralConfig, keygen};
+use rand::{RngCore, SeedableRng};
+use rand_hc::Hc128Rng;
+#[derive(Debug, Parser)]
```

Comment:
Nit: Add empty line and rustdoc.

```suggestion

/// Keygen config.
#[derive(Debug, Parser)]
```